docker-compose.yml

```diff
@@ -70,6 +68,8 @@ services:
       RSSHCOOL_AWS_REGION: ${RSSHCOOL_API_AWS_REGION}
       RSSHCOOL_API_AWS_REST_API_URL: ${RSSHCOOL_API_AWS_TASK_API_URL}
       RSSHCOOL_API_AWS_REST_API_KEY: ${RSSHCOOL_API_AWS_TASK_API_KEY}
+      RSSHCOOL_API_USERS_CLOUD_USERNAME: ${RSSHCOOL_API_VERIFICATION_USERNAME}
```

Comment:
let's remove `_API_`. This extension was added when we had multiple repos long time ago. I'm trying to get rid of it in nest
`RSSHCOOL_USERS_CLOUD_USERNAME` / `RSSHCOOL_USERS_CLOUD_PASSWORD`